src/modules/Dashboard/features/Applet/Schedule/Schedule.tsx

```diff
@@ -48,6 +51,8 @@ export const Schedule = () => {
     };
   }, [appletId, respondentId, ownerId]);
```

Comment:
it seems there is no need in ownerId in dependencies anymore

Reply:
done